specification/purview/data-plane/Azure.Analytics.Purview.Scanning/stable/2023-09-01/scanningService.json

```diff
@@ -0,0 +1,9531 @@
+            "Enabled",
+            "Disabled"
+          ],
+          "type": "string",
+          "description": "Status of versioned scan ruleset properties.",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ScanRulesetStatus",
+            "modelAsString": true
+          }
+        },
+        "version": {
+          "format": "int32",
+          "type": "integer",
+          "description": "Version of versioned scan ruleset properties.",
+          "readOnly": true
+        }
+      }
+    },
+    "ScanRuleset": {
+      "description": "The scan ruleset.",
+      "required": [
+        "kind"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/VersionedScanRuleset"
+        }
+      ],
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/DataSourceType"
+        }
+      },
+      "discriminator": "kind"
+    },
+    "ScanRulesetList": {
+      "description": "List of scan rulesets.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of scan rulesets.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ScanRuleset"
+          }
+        },
+        "nextLink": {
+          "description": "The link to the next page of results, if any remaining results exist.",
+          "type": "string"
+        },
+        "count": {
+          "description": "The number of scan rulesets.",
+          "format": "int64",
+          "type": "integer"
+        }
+      }
+    },
+    "SystemScanRuleset": {
+      "description": "The system scan ruleset.",
+      "required": [
+        "kind"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/VersionedScanRuleset"
+        }
+      ],
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/DataSourceType"
+        }
+      },
+      "discriminator": "kind"
+    },
+    "SystemScanRulesetList": {
+      "description": "List of system scan rulesets.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of system scan rulesets.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SystemScanRuleset"
+          }
+        },
+        "nextLink": {
+          "description": "The link to the next page of results, if any remaining results exist.",
+          "type": "string"
+        },
+        "count": {
+          "description": "The number of system scan rulesets.",
+          "format": "int64",
+          "type": "integer"
+        }
+      }
+    },
+    "RecurrenceScheduleOccurrence": {
+      "description": "The occurrence of recurrence schedule.",
+      "type": "object",
+      "properties": {
+        "additionalProperties": {
+          "type": "object"
+        },
```

Comment:
What additional properties can be present here? This can be passed as input so can the user put _anything_ here? If they do, what is done with it? Please add a description here to explain.

Reply:
Just removed this additionalProperties object. It was a legacy dictionary object in the RecurrenceSchedule object that shouldn't exist anymore.